From 71ae6cc80ee95dda817f1fa15fb764efae61f94c Mon Sep 17 00:00:00 2001
From: Felix Radensky <felix.r@variscite.com>
Date: Thu, 30 Dec 2021 12:08:30 -0800
Subject: [PATCH] imx8mp: Remove uart2 domain restrictions i.MX8MP ATF code
 assigns UART2 to the A53 Domain. Remove this assignment to allow for usage in
 any domain (e.g. M7).

Signed-off-by: Harshesh Valera <harshesh.v@variscite.com>
---
 plat/imx/imx8m/imx8mp/imx8mp_bl31_setup.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/plat/imx/imx8m/imx8mp/imx8mp_bl31_setup.c b/plat/imx/imx8m/imx8mp/imx8mp_bl31_setup.c
index 54fde66d6..7b7133852 100644
--- a/plat/imx/imx8m/imx8mp/imx8mp_bl31_setup.c
+++ b/plat/imx/imx8m/imx8mp/imx8mp_bl31_setup.c
@@ -62,7 +62,6 @@ static const struct imx_rdc_cfg rdc[] = {
 
 
 	/* peripherals domain permission */
-	RDC_PDAPn(RDC_PDAP_UART2, D0R | D0W),
 	RDC_PDAPn(RDC_PDAP_WDOG1, D0R | D0W),
 	RDC_PDAPn(RDC_PDAP_RDC, D0R | D0W | D1R),
 
-- 
2.25.1

